static/src/javascripts/projects/common/modules/identity/modules/switchboardLabel.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+import fastdom from 'lib/fastdom-promise';
+
+export const addSpinner = (labelEl: HTMLElement): Promise<any> =>
+    fastdom.write(() => {
+        if (document.body) {
+            document.body.classList.add('is-updating-cursor');
+        }
+        labelEl.classList.add('is-updating');
+    });
+
+export const removeSpinner = (labelEl: HTMLElement): Promise<any> =>
+    fastdom
+        .write(() => {
+            if (document.body) {
+                document.body.classList.remove('is-updating-cursor');
+            }
+            labelEl.classList.add('is-just-updated');
+            labelEl.classList.remove('is-updating');
+        })
+        .then(
+            () =>
+                new Promise((accept: Function) => {
```

Comment:
For `accept`'s type, can we go with `: () => void`? [The `Function` type is unsafe](https://flow.org/en/docs/types/functions/#toc-function-type) and should be avoided.